templates/downloads.html

```diff
@@ -62,4 +62,13 @@ <h2>Nightly Development Builds</h2>
         <p>Previous nightly builds can be found <a href="https://sourceforge.net/projects/mantid/files/Nightly/">here</a>.</p>
       </article>
     </section>
+        <section id="gridcontainer">
+      <article id="other-software" class="column">
+        <h2>Related Software</h2>
```

Comment:
I know I suggested Related but could we call this "Other Software" and have the description read "Other software relating to neutron data reduction/analysis"?

Reply:
Would it also be difficult to move this to the right-hand side under the nightly downloads?

Reply:
Both are done :+1: